plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2933,3 +2933,42 @@ function wc_update_950_tracking_option_autoload() {
 	);
 	wp_set_option_autoload_values( $options );
 }
+
+/**
+ * Modifies PKs for woocommerce_order_itemmeta and wp_wc_orders_meta tables for more robust select operations.
+ */
+function wc_update_970_modify_primary_key_for_order_related_meta_tables() {
+	global $wpdb;
+
+	$prefix  = $wpdb->prefix;
+	$changes = array(
+		// NEXT: contradicts with the function name a bit.
+		'options'                    => array(
```

Comment:
Benchmarking (5000 entries in the table):
Write 0.000373 -> 0.000375 seconds
Read 0.000193 -> 0.000187 seconds

Not much value honestly.